src/airflow_tools/filesystems/filesystem_factory.py

```diff
@@ -25,6 +34,9 @@ def get_data_lake_filesystem(connection: Connection) -> FilesystemProtocol:
         elif connection.conn_type == "aws":
             hook = S3Hook(aws_conn_id=connection.conn_id)
             return S3Filesystem(hook)
+        elif connection.conn_type == 'google_cloud_platform':
```

Comment:
Just curious, Why do we use `aws`  for `Amazon Web Services` but for `Google Cloud Platform` we don't use `gcp`?

Reply:
We have to use the `conn_type` of the hook we are using:
[See AWS](https://github.com/apache/airflow/blob/main/providers/src/airflow/providers/amazon/aws/hooks/base_aws.py#L476)
[See GCS](https://github.com/apache/airflow/blob/main/providers/src/airflow/providers/google/common/hooks/base_google.py#L218)